input/resources/OperationDefinition-evaluate.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OperationDefinition
+    xmlns="http://hl7.org/fhir">
+    <id value="evaluate"/>
+    <url value="http://hl7.org/fhir/us/davinci-deqm/OperationDefinition/evaluate"/>
+    <name value="Evaluate"/>
+    <title value="Evaluate Operation"/>
+    <status value="active"/>
+    <kind value="operation"/>
+    <publisher value="Clinical Quality Information Workgroup"/>
+    <contact>
+      <telecom>
+        <system value="url"/>
+        <value value="http://www.hl7.org/Special/committees/cqi/index.cfm"/>
+      </telecom>
+    </contact>
+    <description value="The evaluate operation is used to calculate an eMeasure and obtain the results."/>
+    <jurisdiction>
+      <coding>
+        <system value="urn:iso:std:iso:3166"/>
+        <code value="US"/>
+        <display value="United States of America"/>
+      </coding>
+    </jurisdiction>
+    <code value="evaluate"/>
+    <resource value="Measure"/>
+    <system value="false"/>
+    <type value="true"/>
+    <instance value="true"/>
+    <parameter>
+        <name value="periodStart"/>
+        <use value="in"/>
+        <min value="1"/>
+        <max value="1"/>
+        <documentation value="The start of the measurement period. In keeping with the semantics of the date parameter used in the FHIR search operation, the period will start at the beginning of the period implied by the supplied timestamp. E.g. a value of 2014 would set the period start to be 2014-01-01T00:00:00 inclusive"/>
+        <type value="date"/>
+    </parameter>
+    <parameter>
+        <name value="periodEnd"/>
+        <use value="in"/>
+        <min value="1"/>
+        <max value="1"/>
+        <documentation value="The end of the measurement period. The period will end at the end of the period implied by the supplied timestamp. E.g. a value of 2014 would set the period end to be 2014-12-31T23:59:59 inclusive"/>
+        <type value="date"/>
+    </parameter>
+    <parameter>
+        <name value="manifest"/>
+        <use value="in"/>
+        <min value="0"/>
+        <max value="1"/>
+        <targetProfile value="http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-manifestlibrary"/>        
```

Comment:
target profile should be CQFMContentRelease which derives from CRMI manifest.  See https://jira.hl7.org/browse/FHIR-45309 not yet applied

Reply:
This did not make it into PR144, perhaps because dependency is on QMIG ballot-2?